catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -1149,4 +1173,16 @@ private static boolean processEntry(String entryName, List<String> requiredAttrs
 
         return dataIsValid.get();
     }
+
+    public static String getMetacardId(Metacard metacard) {
+        String id = metacard.getId();
+
+        Attribute origIdAttr = metacard.getAttribute(MetacardVersion.VERSION_OF_ID);
```

Comment:
We want the original metacard id vs the history id. The ID should match what the card originally was identified by in order to let the caller track the data correctly (e.g. if they need to delete something on their side).